### ISSUE TITLE
Replace JQuery tooltips plugin

### ISSUE THREAD

**Issue by binarygit** [openfoodfoundation/openfoodnetwork#9541]
### What we should change and why (this is tech debt)

As we are moving away from Angular and JQuery to stimulus, it is necessary to replace the JQuery tooltip plugin with something new. 

### Context

This was talked about here: https://openfoodnetwork.slack.com/archives/C03QBDHDCSY/p1659443811447199

**Comment by sigmundpetersen** [comment]
Hey @binarygit , was this closed by #9560 ?

**Comment by binarygit** [comment]
Nope. There are many places the plugin is in use right now, so we can actually only remove the Jquery tooltip plugin when all of those tooltips are ported over to stimulus. So, I think this one will stay open for sometime.
